ccd-definition/CaseEventToFields/uploadStandardDirectionsAfterGatekeeping.json

```diff
@@ -1,46 +0,0 @@
-[
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseEventID": "uploadStandardDirectionsAfterGatekeeping",
```

Comment:
mapping removed but still event is creatable:
"CaseEventID": "uploadStandardDirectionsAfterGatekeeping",
    "UserRole": "caseworker-publiclaw-courtadmin",
    "CRUD": "CRU"